### PR TITLE
Switches default random source to deterministic

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This changes the default random source to provide deterministic values
similar to how nanotime and walltime do. This also prevents any worries
about if wasm can deplete the host's underlying source of entropy.

### CODE REVIEW

File: internal/platform/crypto.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package platform
+
+import (
+	"io"
+	"math/rand"
+)
+
+// seed is a fixed seed value for NewFakeRandSource.
+//
+// Trivia: While arbitrary, 42 was chosen as it is the "Ultimate Answer" in
```
Comment (codefromthecrypt) [inline internal/platform/crypto.go:10]: a bit wordy, but I suspect not everyone knew why we have 42 in popping up in source files

**Review by mathetake — APPROVED** [review]
